Code/autopkglib/__init__.py

```diff
@@ -512,6 +512,18 @@ def execute_shell(self):
         else:
             sys.exit(0)
 
+    def load_plist_from_file(plist_path, exception_text="Unable to load plist"):
+        """Load plist from file and return content as dictionary"""
+        if not os.path.exists(plist_path):
+            raise ProcessorError(
+                f"File '{plist_path}' does not exist or could not be read."
+            )
```

Comment:
I know you copied this from what was already present, but what's the point of doing this when you're already trying it and catching exceptions below? If we want different output for different possible scenarios, let's catch the specific exception types. Or just stick with the generic one you have, which I think is sufficient for now.

Reply:
You're right. See 7d9d2e0